backend/models/core/Entity.js

```diff
@@ -109,6 +114,67 @@ class Entity {
           value_double
         })); 
   }
+
+  static async create({entries, columns, slug, entity_type_id}) {
+    const db = new DB();
+
+    let attributesValues = ``, insertAttributesSQL = ``;
+
+    let values =``, insertValuesSql = ``;
+    let dec = columns.length - 1;
+
+    const insertEntitiesSQL = `INSERT into entities (slug, entity_type_id) VALUES (:slug, :entity_type_id)`;
+    
+    const isEntities = await db.exectuteStatement(insertEntitiesSQL, [
+      {name: 'slug', value:{stringValue: slug}},
+      {name: 'entity_type_id', value:{longValue: entity_type_id}},
+    ]);
+    
+    if(isEntities) {
+      
+    let temp = [];
+
+    columns.map((col, i) => {
+      entries[`${col}_type`] === 'text' ? temp.push({value_string: entries[col]}) : null;
+      entries[`${col}_type`] === 'textarea' ? temp.push({value_long_string: entries[col]}) : null;
+      entries[`${col}_type`] === 'float' ? temp.push({value_double: entries[col]}) : null;
+
+      attributesValues += ` ${i > 0 ? ',' : ''}('${col}', (SELECT ID from entities ORDER by id desc limit 1),  '${entries[`${col}_type`]}', ${i+1})`;
+      values += ` ${i > 0 ? ',' : ''}((SELECT ID from entities ORDER by id desc limit 1), (SELECT ID from attributes ORDER by id desc limit 1)-${dec},${entries[`${col}_type`] === 'text' ? entries[col] : null}, ${entries[`${col}_type`] === 'textarea' ? entries[col] : null}, ${entries[`${col}_type`] === 'float' ? entries[col] : null})`;
+      dec--;
+    })
+
+    insertAttributesSQL = `INSERT into attributes (name, entity_id, type, \`order\`) VALUES ${attributesValues}`;
+    
+    const isAttributes = await db.exectuteStatement(insertAttributesSQL, []);
+
+    if(isAttributes) {
+      insertValuesSql = `INSERT into \`values\` (entity_id, attribute_id, value_string, value_long_string, value_double) VALUES ${values}`;
+      const isValues = await db.exectuteStatement(insertValuesSql, []);
+    }
+
+    }
+
+    return true;
+  }
+
+  static async delete({id}) {
+    const db = new DB();
+    const deleteEntitiesSQL = 'DELETE from entities where id = :id'
+    const deleteAttributesSQL = 'DELETE from attributes where entity_id = :id'
+    const deleteValuesSQL = 'DELETE from \`values\` where entity_id = :id'
+
+    let executeStatementParam = [
+      {name: 'id', value:{stringValue: id}}
+    ]
+    await db.exectuteStatement(deleteEntitiesSQL, executeStatementParam);
+    await db.exectuteStatement(deleteAttributesSQL, executeStatementParam);
+    await db.exectuteStatement(deleteValuesSQL, executeStatementParam);
```

Comment:
We need to wrap this inside a transaction. I'll update `DB.js` to have a transaction feature.